### ISSUE TITLE
Can't load script data: RemoteData.UserData

### ISSUE THREAD

**Issue by reach2rv** [volkanceylan/Serenity#5257]
Hi @volkanceylan 

I am facing the following issue only after updating to asp.net 3.1 with 2.2 it works without any issues. you may find this issue as a duplicate of #4969  but I have already asked a question in the previously closed issue and still not received any answer. 

```JavaScript
Uncaught Error: Can't load script data: RemoteData.UserData!
    at Object.ensure (Q.ScriptData.ts:114)
    at Object.getRemoteData (Q.ScriptData.ts:177)
    at Object.get (Q.Authorization.ts:63)
    at Object.hasPermission (Q.Authorization.ts:13)
    at QuotationGrid.EntityGrid.hasInsertPermission (EntityGrid.ts:257)
    at Object.disabled (EntityGrid.ts:148)
    at HTMLDivElement.<anonymous> (Toolbar.ts:180)
    at HTMLDivElement.dispatch (jquery-3.5.1.js?v=I8fF0tExdQjoB6bH93fW7Q:5429)
    at HTMLDivElement.$event.dispatch (jquery.event.drag.js?v=Q42_cWjms19Czv7rSFThUg:380)
    at HTMLDivElement.elemData.handle (jquery-3.5.1.js?v=I8fF0tExdQjoB6bH93fW7Q:5233)
```

This precisely happens when hasPermission is called when the module's entity grid is loading. for RemoteUserData.js I can see Error 403 forbidden in Chrome Dev console.

Following are the Code files
![Autho](https://user-images.githubusercontent.com/16520558/96655577-7de45600-135b-11eb-8ec3-a81c9e40e0ac.png)
![getUD](https://user-images.githubusercontent.com/16520558/96655580-7fae1980-135b-11eb-8532-665f6fa01a5c.png)
![ScriptUD](https://user-images.githubusercontent.com/16520558/96655581-8046b000-135b-11eb-864b-eb0b37bd6fa9.png)
![ScriptUDts](https://user-images.githubusercontent.com/16520558/96655583-8046b000-135b-11eb-8ef4-55e6e061fa26.png)
![UDgetFirst](https://user-images.githubusercontent.com/16520558/96655584-80df4680-135b-11eb-9cc6-c3bad6a5ec02.png)
![UDts](https://user-images.githubusercontent.com/16520558/96655586-80df4680-135b-11eb-8c15-bc8e087e7698.png)

Looking forward for your kind support.

PS: I have followed Asp.net update guide written by you step by step without any errors.
